packages/AdobeClient/src/AdobeClientSideKit.js

```diff
@@ -170,7 +170,7 @@ import { AdobeHbkConstructor } from '../../../HeartbeatKit/dist/AdobeHBKit.esm.j
             appMeasurement.timestamp = timestampOption ? Math.floor((new Date).getTime()/1000) : null;
             appMeasurement.events = '';
 
-            if (isInitialized) {
+            if (isClientInitialized) {
```

Comment:
lower down (i can't comment on a portion that wasn't modified),  can you remove the else statement in lines 202-204. this was the misleading lines that logged something not correctly

```
else {
    return 'Error logging event or event type not supported - ' + reportEvent.error;
}
```